src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -362,7 +364,46 @@ export const addForm4142 = formData => {
   return clonedData;
 };
 
+export const delete0781FormData = formData => {
+  // Workaround to avoid eslint rule around mutating params/args
+  const data = formData;
+  delete data.events;
+  delete data.eventTypes;
+  delete data.workBehaviors;
+  delete data.otherBehaviors;
+  delete data.healthBehaviors;
+  delete data.workBehaviorChanges;
+  delete data.socialBehaviorChanges;
```

Comment:
These 2 attributes, `workBehaviorChanges` and `socialBehaviorChanges`, as well as `additionalRemarks781` on L385 are part of the old 0781 and **should not** be deleted.